cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1229,7 +1229,7 @@ request.ui_dispatch_view = search
 search = | tstats `cs_summariesonly_network_traffic` sum(All_Traffic.bytes) as total_bytes, sum(All_Traffic.packets) as total_packets from datamodel=Network_Traffic where `cs_private_ips(All_Traffic.dest_ip)` AND `cs_public_ips(All_Traffic.src_ip)` AND All_Traffic.action="allowed" by All_Traffic.src_ip, All_Traffic.dest_ip, All_Traffic.dest_port, All_Traffic.action \
 | `drop_dm_object_name(All_Traffic)` \
 | join dest_ip dest_port \
-    [| inputlookup cs_all_vuln where (status="open" OR status="reopened") AND port!="0" \
+    [| inputlookup cs_all_vuln where (status="open" OR status="reopened") AND port!="0" AND (severity="critical" OR severity="high" OR severity="medium") \
```

Comment:
Let's keep low in here for now. And once we see many false positives, we can remove it.